src/components/Ticks.tsx

```diff
@@ -78,7 +77,7 @@ export function Tick(props: TickProps) {
         x={textX1}
         y={textY1}
         textAnchor={anchor}
-        alignmentBaseline={alignment}
+        dominantBaseline="middle"
```

Comment:
If you look at the bottom axis, the texts are lower, is it for this change?

Reply:
As said in my previous comment, this is already on master